apps/archive/archive.py

```diff
@@ -1275,6 +1275,13 @@ def _send_mark_user_notifications(
         :param data: kwargs
         """
 
+        user = get_user()
```

Comment:
it should check users from user list, not the current user imo

Reply:
got it